src/applications/healthcare/questionnaire/containers/ConfirmationPage.jsx

```diff
@@ -0,0 +1,69 @@
+import React from 'react';
+import moment from 'moment';
+import { connect } from 'react-redux';
+import Scroll from 'react-scroll';
+
+import { focusElement } from 'platform/utilities/ui';
+
+const scroller = Scroll.scroller;
+const scrollToTop = () => {
+  scroller.scrollTo('topScrollElement', {
+    duration: 500,
+    delay: 0,
+    smooth: true,
+  });
+};
+
+export class ConfirmationPage extends React.Component {
+  componentDidMount() {
+    focusElement('.schemaform-title > h1');
+    scrollToTop();
+  }
+
+  render() {
+    const { submission, data } = this.props.form;
+    const { response } = submission;
+    const name = data.veteranFullName;
+
+    return (
+      <div>
+        <h3 className="confirmation-page-title">Claim received</h3>
+        <p>
+          We usually process claims within <strong>a week</strong>.
+        </p>
+        <p>
+          We may contact you for more information or documents.
+          <br />
+          <i>Please print this page for your records.</i>
+        </p>
+        <div className="inset">
+          <h4>
+            Healthcare Questionnaire Claim{' '}
+            <span className="additional">(Form HC-QSTNR)</span>
+          </h4>
+          <span>
+            for {name.first} {name.middle} {name.last} {name.suffix}
+          </span>
+
+          {response && (
```

Comment:
You may have to check for length as an empty array will always return truthy

Reply:
or use Array.isEmpty

Reply:
Good point! The confirmation will be completely in a separate PR